### PR TITLE
specifies column widths for t1 tracker table

### PR DESCRIPTION
**Pull request by geoffwasham**
@zeue @The-Kergan

### CODE REVIEW

File: docs/tools/t1_coronavirus_symptom_tracker.md [hunk 1]
```diff
@@ -58,13 +58,13 @@ Print out and complete one row per day for each person in your household. For ad
...
-        <th></th>
-        <th></th>
-        <th></th>
+        <th style="width:16%;">**Date /  Name**</th>
+        <th style="width:14%;"></th>
+        <th style="width:14%;"></th>
+        <th style="width:14%;"></th>
+        <th style="width:14%;"></th>
+        <th style="width:14%;"></th>
+        <th style="width:14%;"></th>
```
Comment (zeue) [inline docs/tools/t1_coronavirus_symptom_tracker.md]: I recommend avoiding absolute values and instead use automatic "best fitting" table dimensions, source: https://css-tricks.com/almanac/properties/t/table-layout/

Comment (The-Kergan) [reply docs/tools/t1_coronavirus_symptom_tracker.md]: @zeue great guidance - my approval was rejected. How do I 'unapprove'?

**Comment by geoffwasham** [comment]
Table layout set to fixed, percentage column widths removed

**Comment by The-Kergan** [comment]
@zeue can you please review/amend and approve whatever is needed here? Thanks.

**Comment by zeue** [comment]
@The-Kergan Why are we approving this? It's not working.

**Pull request merged**